File: src/codes/Users_Manual/link/remote_soem_0.cs
~using System.Net;
~using AUTD3Sharp;
using AUTD3Sharp.Link;

~using AUTD3Sharp.Gain;
~using AUTD3Sharp.Modulation;
~using AUTD3Sharp.Utils;
~var autd = Controller.Builder([new AUTD3(Vector3.Zero)]).Open(
RemoteSOEM.Builder(new IPEndPoint(IPAddress.Parse("172.16.99.104"), 8080))
~);

File: src/codes/Users_Manual/link/remote_soem_0.cpp
//~#include<autd3.hpp>
#include "autd3/link/soem.hpp"

//~int main() {
//~auto autd = autd3::ControllerBuilder({autd3::AUTD3(autd3::Vector3::Zero())})
//~                .open(
autd3::link::RemoteSOEM::builder("172.16.99.104:8080")
    //~                    );
    //~return 0; }